### DISCUSSION TITLE
Which key sections (keymaps)

### DISCUSSION THREAD

**Discussion by anotherhadi** [nix-community/nixvim#2265] [Q&A] [answered]
Hey! I want to make subsections for Which key without any actions. For example, on <leader> press, which-key will display the letter "u" as "UI" but as a folder, if I press u I should just see more keymaps.
How can I do this?

**Comment by MattSturgeon** [comment]
I think you want which-key 's `group` option described in its key spec docs.

**Reply by anotherhadi** [reply]
Yes that's it! However, there's no "settings" option for this nixvim plugin, should I put those settings in extraLua or could we add those options in nixvim?

**Reply by MattSturgeon** [reply]
Are you using the stable branch (24.05)? The unstable branch has a freeform settings option.

**Reply by anotherhadi** [reply]
> Are you using the stable branch (24.05)? The unstable branch has a freeform settings option.

Unstable, I look into it then! Ty

**Reply by MattSturgeon** [reply, marked as answer]
The unstable docs are [here](https://nix-community.github.io/nixvim/plugins/which-key/index.html), but note that the way our docs work is that options with sub-options end up getting their own page. You can view that as "nested" pages in the sidebar menu tree.

`plugins.which-key.settings` is documented [here](https://nix-community.github.io/nixvim/plugins/which-key/settings/index.html).
